types/node/events.d.ts

```diff
@@ -106,6 +106,7 @@ declare module "events" {
     type Args<K, T> = T extends DefaultEventMap ? AnyRest : (
         K extends keyof T ? T[K] : never
     );
+    type EventMapKey<T> = T extends DefaultEventMap ? string | symbol : keyof T;
```

Comment:
Is there a reason why this is needed instead of just using the existing `Key<K,T>`?